### ISSUE TITLE
CapabilityStatement.rest.resource.supportedProfiles sollte extrahiert werden

### ISSUE THREAD

**Issue by alexzautke** [medizininformatik-initiative/mii-fhir-site-report-aktin#26]
Zu beachten ist, dass das Feld einen Wert vom Datentyp "Canonical" enthält. Die Version getrennt durch "|" sollte auch übernommen werden.

**Comment by alexanderkiel** [comment]
@alexzautke Wäre es für Dich ein Requirement, dass der FHIR Server unter `CapabilityStatement.rest.resource.supportedProfiles` die KDS Profile aufführt? Blaze macht das z.Z. noch nicht.

**Comment by alexzautke** [comment]
@alexanderkiel Ja, im Modul Person (das einzige Modul das derzeit ein CapabilityStatement enthält) wird dies so gefordert. Dies entspricht auch den Vorgaben aus ISiK an denen wir uns hier orientiert haben.
